### ISSUE TITLE
Creating a README summarizing the Vision, Mission, and Principles

### ISSUE THREAD

**Issue by jpitts** [gitcoinco/ERC-1789#47]
Create a README.md file which summarizes the key attributes of the initiative and proposed governance and funding mechanisms.

This will be placed in a new repo, probably called "docs" or "Start Here". This repo could also become a static-generated TL;DR type of website.

The content work is currently happening in this Google doc, please add comments and edit there:

https://docs.google.com/document/d/1h_-xkEeno7jLx9LddUm_dVvPsZIHgnn7z2gMx2br8sw/edit

**Comment by jpitts** [comment]
A docs repository has been created. The active work is still in the Google doc. Once we're reached consensus on it we'll start to generate the following MD files in docs:

vision-mission-principles.md
block-reward-mechanism-principles.md
governance-principles.md

**Comment by lirazsiri** [comment]
Thank you James for taking initiative this but unfortunately I suspect it's going to be very hard to come up with an internally consistent vision of the mission and principles by using this process. You're trying to define the very soul of governance using a design by committee process that is vulnerable to byzantine actors and sybil attacks. At best the result will be a messy patchwork of half baked ideas.

It's hard enough to coordinate a group of people that are aligned in terms of fundamental driving values and goals. It's impossible when there's no alignment and it's impossible for alignment to exist when what is driving the disalignment are conflicts of interest.

On one hand you have the fakers - fake core "developers" who are mainly interested on developing their personal brand and committing to Twitter. These self promoters can't get enough of the spotlight and media attention so they generate controversy and seed mistrust while paying superficial lip service to community values. The fakers campaign for inflation treasury distribution mechanisms designed to put either themselves or their friends into positions of power in the name of "decentralization".

Then you have mostly neutral facilitators who are trying to avoid taking sides and instead merely create a safe space for everyone to contribute to the discussion.
 
The problem is that the most important people are mostly absent from this political process. They are the closest thing to real "core devs" the community has. They're introverts that hate the spotlight and are mostly too busy solving hard research & engineering problems to be bothered to play politics and win Twitter. They are quietly focused on the work itself and don't want to waste time debating the fakers. "Never wrestle with a pig. You both get dirty and the pig likes it". I assume what the doers want is more funds going to doing.

Sadly, this means the most important voice is usually not heard. If we merely summarize or attempt to average out opinions of the loudest participants in the "governance" and "social consensus" process we'll get a view of consensus that over-represents the fakers and under-represents the doers.

That's the default outcome for the README document.

Even if we could get somehow draft everyone into the debate, I'm skeptical it would be possible to come up with one consistent document that both the doers and fakers would be happy with.

**Comment by jpitts** [comment]
>  If we merely summarize or attempt to average out opinions of the loudest participants in the "governance" and "social consensus" process we'll get a view of consensus that over-represents the fakers and under-represents the doers.

I definitely have this in mind, employing the best filters I can muster! "Fakers are gonna fake", therefore we have to be careful to not create a system which empowers charlatans. Part of the cure is to expect good-faith participation and the highest quality possible. We should definitely be critical thinkers in this effort. 

> Even if we could get somehow draft everyone into the debate, I'm skeptical it would be possible to come up with one consistent document that both the doers and fakers would be happy with.

I see the process as emergent. Take the early ideas and long-form proposals, set down some very basic principles based on those. Those principles will guide the next iteration. 

A crisis occurs when it is seen that a principle hampers progress, so then we resolve that, knowing who has expertise and leadership ability to help guide us forward.

**Comment by jpitts** [comment]
The results of this work is now placed in the docs repo:
https://github.com/ethereum-funding/docs

These are the specific documents created:
[Vision, Mission, Principles](https://github.com/ethereum-funding/docs/blob/master/vision-mission-principles.md)
[Block Rewards Mechanism](https://github.com/ethereum-funding/docs/blob/master/block-rewards-mechanism-design-principles.md)
[Fund Allocation](https://github.com/ethereum-funding/docs/blob/master/fund-allocation-governance.md)

**Comment by lrettig** [comment]
> The problem is that the most important people are mostly absent from this political process... They're introverts that hate the spotlight and are mostly too busy solving hard research & engineering problems to be bothered to play politics and win Twitter

I agree strongly with this sentiment, and I think it nicely captures some of the biggest challenges we're facing in Ethereum governance (and probably in the wider world as well). But I strongly disagree that, just because some respected voices opt out of the conversation, that we should cease all attempts to discuss governance or find common ground. If we do this right, maybe some of those voices will change their minds and decide to contribute, and we should always make space for them if they do. We should find ways for them to participate that are comfortable for them. And if they choose not to, that's okay too - from my perspective one of the main goals of funding specifically and of governance more generally is to enable the builders to keep building.
